src/utils.cc

```diff
@@ -775,3 +775,36 @@ uint32_t hash_file(const std::string &filename)
 
 	return out;
 }
+
+int look_path(const std::string &file, std::string *out)
+{
+    const char *sys_path = getenv("PATH");
+	if (!sys_path)
+		return -1;
+
+    const std::vector<std::string> sys_paths = split_string(sys_path, ":");
```

Comment:
This looks like a mix of tabs and space, or otherwise the indentation is strange?

Reply:
I configured my editor to use space for indentation.
When I found that `kcov` code use tabs, I added an `.editorconfig` file with the correct style, but since I updated the code manually some line where not formatted correctly.

It seems that the same problem is also in other functions.

Reply:
Right. I've started outs with tab indentation, but I've also later come to prefer spaces, so the code base is probably mixed.

I guess I should reformat it all sometime in the future...